user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -1314,4 +1322,24 @@ function commonFunctions:pathJoin(...)
   return table.concat(args, "/")
 end
 
+-- ---------------------------------------------------------------------------------------------
+--28. Function gets parameter value from CMakeCache.txt file
```

Comment:
@aderiabin Please remove this function since it's not required anymore

Reply:
@dboltovskyi Done in e1265d7